Doc/c-api/module.rst

```diff
@@ -486,12 +486,29 @@ state:
    .. versionadded:: 3.10
 
 
+.. c:function:: int PyModule_Add(PyObject *module, const char *name, PyObject *value)
+
+   Similar to :c:func:`PyModule_AddObjectRef`, but "steals" a reference
+   to *value*.
```

Comment:
PyModule_AddObjectRef() must not be called with NULL. Can you please elaborate the behavior when value is NULL? Something like:

> If value is NULL, just return -1.

Reply:
`PyModule_AddObjectRef()` can be called with NULL if an exception is set. Actually, some of my fixes which use `PyModule_AddObjectRef()` rely on this.